cdap-unit-test/src/test/java/io/cdap/cdap/test/base/TestFrameworkExploreTestSuite.java

```diff
@@ -38,13 +32,7 @@
 @Category(XSlowTests.class)
 @RunWith(TestSuite.class)
 @Suite.SuiteClasses({
-  DummyBaseTestRun.class,
-  DummyBaseCloneTestRun.class,
   DynamicPartitioningTestRun.class,
```

Comment:
What is this last test testing? From description all of this seems to be related to explore

Reply:
Removed it